crates/indexer/src/processors/default_processor.rs

```diff
@@ -117,8 +117,7 @@ fn insert_transactions(
             diesel::insert_into(schema::transactions::table)
                 .values(&txns[start_ind..end_ind])
                 .on_conflict(version)
-                .do_update()
-                .set((epoch.eq(excluded(epoch)),)),
```

Comment:
why not upsert?

Reply:
good catch i missed this one. should be .do_nothing b/c we don't want to keep triggering reprocessing of the same row in stitch.

Reply:
If we actually need to change something we should trigger a backfill job and modify the particular column that needs to be changed. Not just override the old row by default it could lead to bad stuff.